changelog/unreleased/kong/dynamic-log-level-rpc.yml

```diff
@@ -0,0 +1,6 @@
+message: |
+  Dynamic log level over Hybrid mode RPC which allows setting DP log level
```

Comment:
Are we sure this PR contains this RPC? This was supposed to be dead.

Reply:
We  will remove it, KAG-5574.